localstack/services/apigateway/provider.py

```diff
@@ -644,14 +657,13 @@ def import_rest_api(
             CreateRestApiRequest(name=openapi_spec.get("info").get("title")),
         )
 
-        return _call_moto(
+        return self.put_rest_api(
```

Comment:
Out of curiosity - was this change required? The `_call_moto(..)` util function is performing a few internal remappings, creating a new local `context`, etc. We would lose that context remapping/rectification if we call the method directly.

Reply:
Good point, I think we can go directly to moto.

Reply:
Actually, sorry, the previous comment may have been confusing, let me rephrase. I think we **do** need to call the `put_rest_api(..)` method directly, however, ideally with an updated `context` variable. 

Maybe we could pull out the util code `local_context = create_aws_request_context(..)` which creates a custom context inside `_call_moto(..)`, and make it reusable, such that we can pass a proper/updated context here?

Something like:
```
request = PutRestApiRequest(
    restApiId=response.get("id"),
    failOnWarnings=str_to_bool(fail_on_warnings) or False,
    parameters=parameters or {},
    body=io.BytesIO(body_data),
)
local_context = create_custom_context(context, "PutRestApi", request)
return self.put_rest_api(context, request)
```

Reply:
So I went down that path and did the same for creation. The import operation is a combination of create+put. I extracted the function and created a new context. Along the way learned that account_id and the headers need to be set on the new context, something that before wasn't happening on `_call_moto` because it was based on the original context; it adds a bit of minimum overhead but nothing major.

For the record, I don't think the new context was necessary given how the put_rest_api operation is coded since it is not making a call to the backend (moto), so no "dispatching" is happening.